cmd/puller/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"flag"
+	"github.com/kubeflow/kfserving/pkg/puller"
+	"log"
+)
+
+var (
+	modelDir = flag.String("model_dir", "/tmp","directory for multi-model config files")
```

Comment:
Can we default to `/mnt/models` ?

Reply:
These are config files, I thought we would default `/mnt/models` for model files, but configs would go into a different location. Lmk where you prefer